openapi-integration-tests/src/test/java/io/ballerina/openapi/extension/build/BuildExtensionTests.java

```diff
@@ -47,18 +49,17 @@ public void setupDistributions() throws IOException {
     }
 
     @Test(description = "Check openapi build plugin in `bal build` command")
-    public void annotationWithOutBuildOption() throws IOException {
+    public void onlyBuildCommand() throws IOException {
         executeCommand("project_1");
     }
```

Comment:
Shouldn't we have assertions in here to ensure that any open API contracts are not generated in the `target` directory?

Reply:
yeah we need to test, this assertion implementation working on separate PR. :)